libs/ha/ha_common_libs_k8s.py

```diff
@@ -1256,7 +1256,9 @@ def setup_mock_monitor(node_obj):
         return True
 
     def simulate_disk_cvg_failure(self, node_obj, source, resource_status, resource_type,
-                                  node_type='data', resource_cnt=1, node_cnt=1, delay=None):
+                              node_type='data', resource_cnt=1, node_cnt=1, delay=None,
+                              specific_info=False, generation_id=None, node_id=None,
+                              resource_id=None):
```

Comment:
Ids can be extracted at runtime, why  generation_id=None, node_id=None,  resource_id=None are added?

Reply:
By default it will be extracted at run time only, for our convenience to check reverse path from Hare to HA, we need this to be given by the user in order to verify ids are same as that of forward path.